diff --git a/rmk/src/split/peripheral.rs b/rmk/src/split/peripheral.rs
index 65ea96bb..c60066d6 100644
--- a/rmk/src/split/peripheral.rs
+++ b/rmk/src/split/peripheral.rs
@@ -9,6 +9,7 @@ use {super::ble::PeerAddress, crate::channel::FLASH_CHANNEL};
 #[cfg(feature = "_ble")]
 use {
     crate::event::{BatteryStatusEvent, ChargingStateEvent, EventSubscriber},
+    rmk_types::battery::BatteryStatus,
     trouble_host::prelude::*,
 };
 
