diff --git a/rmk/src/keymap.rs b/rmk/src/keymap.rs
index 136347166..9c8510278 100644
--- a/rmk/src/keymap.rs
+++ b/rmk/src/keymap.rs
@@ -177,7 +177,6 @@ impl KeyMapInner<'_> {
         if before != after {
             publish_event(LayerChangeEvent::new(after));
         }
-        publish_event(LayerChangeEvent::new(self.get_activated_layer()));
     }
 
     fn get_action_at(&self, pos: KeyboardEventPos, layer_num: usize) -> KeyAction {
