diff --git a/rmk/src/ble/mod.rs b/rmk/src/ble/mod.rs
index aa98c87a1..0193da9df 100644
--- a/rmk/src/ble/mod.rs
+++ b/rmk/src/ble/mod.rs
@@ -186,6 +186,7 @@ impl<
             run_ble_keyboard(
                 &stack,
                 &self.device_config,
+                self.ble_name,
                 &self.config,
                 #[cfg(feature = "host")]
                 self.host_service,
diff --git a/rmk/src/host/rynk/mod.rs b/rmk/src/host/rynk/mod.rs
index 857d7165a..620736c13 100644
--- a/rmk/src/host/rynk/mod.rs
+++ b/rmk/src/host/rynk/mod.rs
@@ -169,6 +169,8 @@ impl<'a> RynkService<'a> {
             Cmd::ClearBleProfile => Some(true),
             #[cfg(all(feature = "_ble", feature = "split"))]
             Cmd::SetSplitCentralLatency => Some(true),
+            #[cfg(feature = "split")]
+            Cmd::SetSplitTransportForce => Some(true),
             #[cfg(feature = "_ble")]
             Cmd::SetBleName => Some(true),
             Cmd::SetKeyAction
@@ -265,7 +267,7 @@ impl<'a> RynkService<'a> {
             #[cfg(feature = "_ble")]
             Cmd::GetBleStatus | Cmd::SwitchBleProfile | Cmd::GetBatteryStatus | Cmd::GetBleName => Some(false),
             #[cfg(feature = "split")]
-            Cmd::GetPeripheralStatus => Some(false),
+            Cmd::GetPeripheralStatus | Cmd::GetSplitTransport => Some(false),
             #[cfg(all(feature = "_ble", feature = "split"))]
             Cmd::GetSplitCentralLatency => Some(false),
             #[cfg(feature = "lighting")]
