diff --git a/rmk/src/ble/mod.rs b/rmk/src/ble/mod.rs
index 840e749a7..4a01f99dd 100644
--- a/rmk/src/ble/mod.rs
+++ b/rmk/src/ble/mod.rs
@@ -201,6 +201,7 @@ impl<
             run_ble_keyboard(
                 &stack,
                 &self.device_config,
+                self.ble_name,
                 &self.config,
                 #[cfg(feature = "host")]
                 self.host_service,
diff --git a/rmk/src/host/rynk/mod.rs b/rmk/src/host/rynk/mod.rs
index 710ae8edc..468d52217 100644
--- a/rmk/src/host/rynk/mod.rs
+++ b/rmk/src/host/rynk/mod.rs
@@ -257,7 +257,9 @@ impl<'a> RynkService<'a> {
             #[cfg(feature = "_ble")]
             Cmd::GetBleStatus | Cmd::SwitchBleProfile | Cmd::GetBatteryStatus => Some(false),
             #[cfg(feature = "split")]
-            Cmd::GetPeripheralStatus => Some(false),
+            Cmd::GetPeripheralStatus | Cmd::GetSplitTransport => Some(false),
+            #[cfg(feature = "split")]
+            Cmd::SetSplitTransportForce => Some(true),
             Cmd::SetMorseProfileEntry | Cmd::DeleteMorseProfile => Some(true),
             Cmd::GetBuildInfo
             | Cmd::GetMorseHoldTriggerPositions
